### PR TITLE
feat(router): nginx 1.9.0. Remove third party tcp module

### PR DESCRIPTION
**Pull request by aledbf**
http://nginx.org/en/docs/stream/ngx_stream_core_module.html

### CODE REVIEW

File: router/image/templates/nginx.conf [hunk 1]
```diff
@@ -292,22 +292,15 @@ http {
...
-
+stream {
     upstream builder {
         server {{ .deis_builder_host }}:{{ .deis_builder_port }};
     }
 
     server {
         listen 2222;
+        proxy_connect_timeout  {{ or (.deis_router_builder_timeout_connect) "10000" }};
+        proxy_timeout          {{ or (.deis_router_builder_timeout_tcp) "1200000" }};
```
Comment (bacongobbler) [inline router/image/templates/nginx.conf]: could you please update the docs to reflect this new value?

**Comment by mboersma** [comment]
Much simpler without having to apply the external patch. Code LGTM.

**Comment by carmstrong** [comment]
LGTM once @bacongobbler's comment is addressed. Nice cleanup.

**Comment by carmstrong** [comment]
Sorry, misclick :disappointed:

File: router/image/templates/nginx.conf (hunk 1 above)
Comment (aledbf) [inline router/image/templates/nginx.conf]: which one? (In fact I removed the send and read builder timeouts)

Comment (mboersma) [inline router/image/templates/nginx.conf]: (done in b5b3b2d332d2501659c993577ab7dd0309184879)

**Comment by mboersma** [comment]
The [commit style](http://docs.deis.io/en/latest/contributing/standards/#commit-style) would be better served with something like:
```feat(router): update to nginx 1.9 for built-in stream module```
Start with a verb, keep it lower-case. Just nitpicking, not necessary. :smiling_imp:

File: router/image/templates/nginx.conf (hunk 1 above)
Comment (carmstrong) [inline router/image/templates/nginx.conf]: I think @bacongobbler was referring to `/deis/router/builder/timeout/tcp`, but it already exists: http://docs.deis.io/en/latest/customizing_deis/router_settings/

**Comment by bacongobbler** [comment]
Yep, I'm all happy :)

**Comment by carmstrong** [comment]
:shipit: @aledbf

**Pull request merged**